components/dashboard-nav.tsx

```
@@ -9,7 +9,7 @@ import { Button } from "@/components/ui/basic/button"
 import { motion } from "framer-motion"
 import { 
   LayoutDashboard, Briefcase, BarChart2, Users, Calendar, 
-  LogOut, Settings, MessageSquare, GraduationCap 
+  LogOut, Settings, Building, GraduationCap 
```

ESLint

[error] 'GraduationCap' is defined but never used.

(@typescript-eslint/no-unused-vars)